config.json

```diff
@@ -215,7 +215,7 @@
       "slug": "prime-factors",
       "uuid": "ee849e23-d709-4e4a-ab89-07a87e13e4fb",
       "core": false,
-      "unlocked_by": "sum-of-multiples",
+      "unlocked_by": null,
```

Comment:
This might be a bit of an issue. We anchored 5 optional exercises to `sum-of-multiples`. Those become free-floating with this change, plus some others.